lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -993,6 +993,41 @@ describe('StartProcessCloudComponent', () => {
 
             expect(card).toBeTruthy();
         });
+
+        it('start process button should be enabled when isLoading is false', async () => {
+            component.ngOnChanges({ appName: firstChange });
+            component.processDefinitionList = fakeProcessDefinitions;
+            component.processDefinitionName = fakeProcessDefinitions[0].name;
+            component.ngOnInit();
```

Comment:
typically `ngOnInit` is not needed

Reply:
Here is needed because in ngOnInit there is a code related to setting process definition name which is used in further evaluation of whether start button should be enabled or not

Reply:
fixed